java/azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/AzureStorageCheckpointLeaseManager.java

```diff
@@ -46,13 +47,21 @@
 
     AzureStorageCheckpointLeaseManager(String storageConnectionString)
     {
-        this.storageConnectionString = storageConnectionString;
+        this(storageConnectionString, null);
     }
-
+    
     AzureStorageCheckpointLeaseManager(String storageConnectionString, String storageContainerName)
     {
+    	this(storageConnectionString, storageContainerName, null);
+    }
+
+    AzureStorageCheckpointLeaseManager(String storageConnectionString, String storageContainerName, String storageBlobPrefix)
+    {
         this.storageConnectionString = storageConnectionString;
         this.storageContainerName = storageContainerName;
+        // Get rid of whitespace. An all-whitespace prefix will end up empty. The rest of the code
+        // needs to be able to handle null (no prefix supplied) or empty (bogus all-whitespace prefix supplied).
+        this.storageBlobPrefix = (storageBlobPrefix != null) ? storageBlobPrefix.trim() : null;
```

Comment:
>(storageBlobPrefix != null) ? storageBlobPrefix.trim() : null; [](start = 33, length = 62)

Why don't you just assign empty string if it is null? This will save you from doing another null check below. After all empty string is idempotent.